tools/get_whitelisting_digest.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+import argparse
+import os
+import sys
+
+# make rpmlint modules available
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
Use apostrophes in this file instead of quotes.

Reply:
I thought I catched all of them double quotes. Why aren't the flake8 warnings showing up in my local test run? Is there something special I need to do?

Reply:
Note flake8 uses plugins, so it's very likely you miss one of these:
https://github.com/rpm-software-management/rpmlint/blob/8cc9efa1065835753cdb99e15a8016391ca9d9d9/.github/workflows/main.yml#L44-L65

Anyway, I'm going to add a test that will want about missing plug-in when pytest is run. That can simplify the development of the tool.